### PR TITLE
Support passing opts to spawn via SubProcess

### PR DESCRIPTION
**Pull request by michaelcontento**

### CODE REVIEW

File: test/subproc-specs.js [hunk 1]
```diff
@@ -40,6 +40,23 @@ describe('SubProcess', () => {
     let x = new SubProcess('ls');
     x.args.should.eql([]);
   });
+  it('should default opts dict to {}', () => {
+    let x = new SubProcess('ls');
+    x.opts.should.eql({});
+  });
+  it('should pass opts to spawn', async () => {
+    const cwd = path.resolve(getFixture('.'));
+    const subproc = new SubProcess('ls', [], { cwd });
```
Comment (jlipps) [inline test/subproc-specs.js]: minor style nitpick, I'd prefer simply `{cwd}` if you don't mind.

Comment (michaelcontento) [inline test/subproc-specs.js]: Fixed

**Comment by jlipps** [comment]
published in 1.5.1

**Comment by michaelcontento** [comment]
Thank you very much! :+1:

**Pull request merged**